### PR TITLE
[MIG][12.0] project_bid

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Standard migration

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@mreficent Please review

File: project_bid/README.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+=====
+
+* Add users to the groups 'Project Bid / User' or 'Project Bid / Manager'
+* Go to 'Project / Configuration / Project Bid Templates' and define standard
+  parameters that will be used across multiple bids.
+* Go to 'Project / Project Bid' to create new Bids.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
```
Comment (MiquelRForgeFlow) [inline project_bid/README.rst]:
```suggestion
   :target: https://runbot.odoo-community.org/runbot/140/12.0
```

File: project_bid/README.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
```
Comment (MiquelRForgeFlow) [inline project_bid/README.rst]:
```suggestion
.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
```

File: project_bid/README.rst [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+   :target: https://www.gnu.org/licenses/lgpl.html
```
Comment (MiquelRForgeFlow) [inline project_bid/README.rst]:
```suggestion
   :target: https://www.gnu.org/licenses/lgpl
```

File: project_bid/README.rst [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (MiquelRForgeFlow) [inline project_bid/README.rst]:
```suggestion
* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
```

File: project_bid/model/project_bid_template.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    _description = "Project Bid Template"
+
+    name = fields.Char("Description", size=256, required=True)
+    default_component_labor = fields.Many2many(
+        "product.product",
+        "project_bid_product_rel",
+        "product_id",
+        "project_bid_template_id",
+        string="Default component labor",
+        required=False,
```
Comment (MiquelRForgeFlow) [inline project_bid/model/project_bid_template.py]: not needed

File: project_bid/view/project_bid_view.xml [hunk 6]
```diff
@@ -302,7 +281,6 @@
     <record id="view_project_bid_tree" model="ir.ui.view">
         <field name="name">project.bid.tree</field>
         <field name="model">project.bid</field>
-        <field name="type">tree</field>
```
Comment (MiquelRForgeFlow) [inline project_bid/view/project_bid_view.xml]: why?

**Comment by MiquelRForgeFlow** [comment]
maybe it's better to remove all info related to OCA

**Comment by AaronHForgeFlow** [comment]
Can you give me a hand plz?

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**